java/ql/test/query-tests/security/CWE-089/semmle/examples/taintedString.expected

```diff
@@ -20,6 +20,9 @@
 | Test.java:29:22:29:28 | tainted | 26 | Test.java:55:22:55:28 | ...[...] |
 | Test.java:29:22:29:28 | tainted | 29 | Test.java:58:4:58:27 | append(...) |
 | Test.java:29:22:29:28 | tainted | 29 | Test.java:58:19:58:26 | category |
+| Test.java:29:22:29:28 | tainted | 30 | Test.java:59:4:59:10 | querySb |
```

Comment:
Could you explain why these results changed? Did you fix a bug somewhere? I'd expect a refactor to be behaviour preserving.

Reply:
I noticed that `StringBuilder.append` wasn't taint preserving to the qualifier, which seemed incorrect so I made it so after refactoring that part.
Most of the time it doesn't matter because there's a special step from arguments of `append` and similar methods to a `toString` method of the same `StringBuilder`, but it could matter if it's passed into a `Formatter`.